src/content/docs/en/concepts/islands.mdx

```diff
@@ -84,4 +84,45 @@ Even better, you can tell Astro exactly how and when to render each component. I
 
 In Astro, it’s up to you as the developer to explicitly tell Astro which components on the page need to also run in the browser. Astro will only hydrate exactly what’s needed on the page and leave the rest of your site as static HTML. 
 
-**Islands are the secret to Astro’s fast-by-default performance story!**
+**Client islands are the secret to Astro’s fast-by-default performance story!**
+
+## Server islands
+
+Each server island is a UI component that is rendered to the page independently from the rest, making it easy to combine high performance static HTML and dynamic server-generated components.
+
+Add the [`server:defer` directive](/en/reference/directives-reference/#server-directives) to any component on your page to turn it into its own island:
+
+```astro title="src/pages/index.astro" "server:defer"
+---
+import Avatar from '../components/Avatar.astro';
+---
+<Avatar server:defer />
+```
+
+This allows your page's critical content to be rendered immediately as a shell, while deferring your most dynamic UI elements, such as user avatars or product reviews. This also means that a slower island, such as one connected to a legacy backend, won’t delay the rest of the page's personalized content from being available.
```

Comment:
Instead of saying "legacy backend", I'd say something about different API endpoints load at different speed and server islands allow them to all load in parallel. I'm afraid that seeing "legacy backend" will make some people think that the feature isn't valuable if you don't have legacy backends.